packages/api/src/templates/locales/es.json

```diff
@@ -92,7 +92,7 @@
     "SUBJECT": "sus documentos de certificación de LiteFarm",
     "HI": "Hola",
     "GET_YOUR_EXPORT": "Obtener documentos",
-    "LANGUAGE_DELAY": "Los cultivos e insumos en el reporte estan en inglés, pero muy pronto estara disponible en español. Pedimos disculpas por los inconvenientes."
```

Comment:
This problem again where English is an empty string because english is supported and this string explains why the other language is not supported. Might need to manually add this back in crowdin or something.

Reply:
So it hasn't been translated in the new languages either then? Yeah we need a better code set up for this. I'll make a ticket.